package.json

```diff
@@ -2,17 +2,17 @@
   "name": "ifrau",
   "version": "0.37.0",
   "description": "Free-range app utility for IFRAME-based FRAs",
-  "main": "src/index.js",
+  "type": "module",
+  "exports": "./index.js",
+  "files": [
+    "host.js",
+    "./client",
+    "./plugins",
+    "./port"
+  ],
   "scripts": {
-    "prebrowserify": "rimraf dist && mkdir dist && cd dist && mkdir ifrau",
-    "browserify": "npm run browserify:full && npm run browserify:client && npm run browserify:host && npm run browserify:host:global",
-    "browserify:client": "browserify -g uglifyify -s ifrau/client ./client > ./dist/ifrau/client.js",
-    "browserify:host": "browserify -g uglifyify -s ifrau/host ./host > ./dist/ifrau/host.js",
-    "browserify:host:global": "browserify -g uglifyify -p deumdify -s ifrau/host ./host > ./dist/ifrau/host-global.js",
-    "browserify:full": "browserify -g uglifyify -s ifrau . > ./dist/ifrau.js",
     "lint": "eslint .",
-    "test:unit": "istanbul cover node_modules/mocha/bin/_mocha -- -R spec --recursive",
-    "pretest": "npm run browserify",
+    "test:unit": "mocha ./test/**/*.test.js",
```

Comment:
We don't need jsdom anymore now that `unique-id` isn't using globals.

Reply:
... I say that as two tests fail because they can't access the window object. It's weird because they pass locally and I wiped out node_modules and re-ran `npm i` after removing the jsdom reference. Oh well.